client/fns_test.go

```diff
@@ -36,29 +39,53 @@ func TestVsockIdPort(t *testing.T) {
 }
 
 func TestParseBinds(t *testing.T) {
+	td := func(s string) string {
+		return path.Join(os.TempDir(), s)
```

Comment:
maybe t.TempDir() will be idiomatic?

Reply:
both were wrong, it has to /tmp :-)
see comment as to why.